Misc/NEWS.d/next/Library/2021-05-31-04-51-02.bpo-43858.r7LOu6.rst

```diff
@@ -0,0 +1 @@
+Added a function that returns a copy of a dict of logging levels: :func:`logging.getLevelNamesMapping()`
```

Comment:
My understanding is that `:func:` adds the parens, so you should leave them off in this message. I've looked at other files, and they don't have the parens.